src/components/Button.tsx

```diff
@@ -14,16 +14,17 @@ export enum ButtonType {
     lightBlue = "lightBlue",
 }
 
-type AletheiaButtonProps = Omit<ButtonProps, "type">;
-interface IAletheiaButtonProps extends AletheiaButtonProps {
-    type?: ButtonType;
+interface IAletheiaButtonProps extends Omit<ButtonProps, "type"> {
```

Comment:
suggestion: I suggest that you don't try to pair the interface with the MUI button in this change. You can keep the AletheiaButton current interface so you don't need to change too many files.